packages/web-api/src/WebClient.ts

```diff
@@ -257,6 +271,13 @@ export class WebClient extends Methods {
     // serializeApiCallOptions will always determine the appropriate content-type
     this.axios.defaults.headers.post['Content-Type'] = undefined;
 
+    // request interceptors have reversed execution order
```

Comment:
That is correct. Previously when `serializeApiCallOptions` was set by [transformRequest](https://github.com/slackapi/node-slack-sdk/pull/2076/files#diff-a395355361c79a96d5caf3f7ac02c9bf16067d13f93da0fe3f68a0d782f075aeL248) it would always be run at the end. This is not the desired behaviour when configuring an interceptor. We would like the custom interceptor to have the final say on the out going request.